tools/QuietBackgroundProcesses/DevHome.QuietBackgroundProcesses.ElevatedServer/QuietBackgroundProcessesSession.cpp

```diff
@@ -61,15 +63,30 @@ namespace ABI::DevHome::QuietBackgroundProcesses
             // Start timer
             g_activeTimer.reset(new TimedQuietSession(duration));
 
+            // Start performance recorder
```

Comment:
>performance recorder [](http://example.com/codeflow?start=21&length=20)

have you measured the performance impact of collecting the trace? Looks like the impact may scale linearly with increasing running processes. May be worth having a checkbox that enabled/disable analytics collection?

Reply:
so... the tracing traces itself :P  (you can see its non-zero impact in its own output)

I'll bring it up with Bridget & Tyler.